From 0dcd9fb64971b75a81d53a6e0e63b9b0d56ebbd2 Mon Sep 17 00:00:00 2001
From: Liming Wang <liming.wang@windriver.com>
Date: Tue, 13 Nov 2012 20:11:08 +0800
Subject: [PATCH 03/26] SDHCI: Change target_phys_addr_t to uint32_t

Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 hw/sdhci.c |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/hw/sdhci.c b/hw/sdhci.c
index b387013..3e61878 100644
--- a/hw/sdhci.c
+++ b/hw/sdhci.c
@@ -500,7 +500,7 @@ static void sdhci_command(SDHCIState *s, uint32_t cmd)
     sdhci_set_irq(s);
 }
 
-static uint32_t sdhci_read32i(void *opaque, target_phys_addr_t offset)
+static uint32_t sdhci_read32i(void *opaque, uint32_t offset)
 {
     SDHCIState *s = opaque;
 
@@ -561,7 +561,7 @@ static uint32_t sdhci_read32i(void *opaque, target_phys_addr_t offset)
     return 0;
 }
 
-static uint64_t sdhci_read(void *opaque, target_phys_addr_t offset,
+static uint64_t sdhci_read(void *opaque, uint32_t offset,
         unsigned size)
 {
     uint32_t ret = sdhci_read32i(opaque, offset & ~0x3);
@@ -571,7 +571,7 @@ static uint64_t sdhci_read(void *opaque, target_phys_addr_t offset,
     return ret;
 }
 
-static void sdhci_write_masked(void *opaque, target_phys_addr_t offset,
+static void sdhci_write_masked(void *opaque, uint32_t offset,
                                uint32_t value, uint32_t mask)
 {
     DB_PRINT("addr=" TARGET_FMT_plx " = %x, mask = %08x\n", offset, value,
@@ -658,7 +658,7 @@ static void sdhci_write_masked(void *opaque, target_phys_addr_t offset,
     }
 }
 
-static void sdhci_write(void *opaque, target_phys_addr_t offset,
+static void sdhci_write(void *opaque, uint32_t offset,
                           uint64_t value, unsigned size)
 {
     uint32_t mask = (1ULL << (size * 8)) - 1;
-- 
1.7.9.5

